packages/pages/src/plugins/index.ts

```diff
@@ -0,0 +1,2 @@
+import urlWritebackPlugin from "./urlWriteback/index.ts";
```

Comment:
cc: @mkilpatrick

Reply:
If they're in their own workspace then it'll be a different npm package which we don't want.

Reply:
Does it make sense for this to be in an npm package at all given that it's intended to be run from Deno? I'm not sure what the use case for importing into a node environment would be.

Reply:
It looks like it's breaking the tests as is because of the `Deno` reference and the different import styles

Reply:
Oic. My other comments was wrong about the import now thinking about it. This will never be imported as an npm module. Given that, I think we can make it its own workspace. It still requires a package.json but it will only be published to deno, not npm.

Unclear if we'll want a whole deno release mechanism for it, but we can figure that out later.